samples/client/petstore/typescript-node/client.ts

```diff
@@ -1,59 +0,0 @@
-import api = require('./api');
```

Comment:
@Vrolijkx when you've time, would you please file another PR to restore this file so that we've something to test the Typescript node petstore?